op-acceptance-tests/tests/interop/loadtest/interop_load_test.go

```diff
@@ -100,15 +122,16 @@ func TestExecFromSameAddressInALoop(gt *testing.T) {
 			includedBlock, err := tx.IncludedBlock.Eval(t.Ctx())
 			t.Require().NoError(err)
 			for {
-				crossSafeID, err := sys.Supervisor.Escape().QueryAPI().CrossSafe(t.Ctx(), sys.L2ChainB.ChainID())
+				// TODO can we use proxyd to see if something is crossafe?
+				crossSafeID, err := sys.Supervisor.Escape().QueryAPI().CrossSafe(t.Ctx(), l2ELB.ChainID())
```

Comment:
Question for reviewers. Querying the supervisor directly may not be ideal, but it could be harmless enough to avoid spending more time on this.

Reply:
I think this is fine for now, @pcw109550 added some helpers in https://github.com/ethereum-optimism/optimism/pull/15908 that might be useful when they merge.